includes/rest/class-following.php

```diff
@@ -98,4 +102,27 @@ public static function request_parameters() {
 
 		return $params;
 	}
+
+	/**
+	 * Add the Blog Authors to the following list of the Blog Actor
+	 * if Blog not in single mode.
+	 *
+	 * @param array $array The array of following urls.
+	 * @param User  $user  The user object.
+	 *
+	 * @return array The array of following urls.
+	 */
+	public static function default_following( $array, $user ) {
+		if ( 0 === $user->get__id() && ! is_single_mode() ) {
```

Comment:
in `functions.php`

It checks if the blog-user is the only user that is enabled. In that case it makes no sense to show the author profiles as followings, because they will not work.

Reply:
oops, typo! thanks for finding this!!!